index.html

```diff
@@ -83,6 +83,10 @@ <h2>Introduction</h2>
         <p>This document attempts to explain what you can do with DIDs and why you might care to use them. We start by describing the basic conceptual framework for DIDs, including basic terminology, followed by the actions one can take with a DID. Then we introduce the defining features and benefits of DIDs along with five focal use cases illustrating those features and benefits. 
         </p>
       </section>
+    <section class="informative">
+        <h1>Terminology</h1>
```

Comment:
This section should be flagged as under discussion, referencing https://github.com/w3c/did-core/issues/4 at a minimum. There's a way in ReSpec to do that automagically.